AccessibilityInsightsForAndroidService/.idea/deploymentTargetDropDown.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we should add this file to `.gitignore` and exclude it from being added

Reply:
This file also seems to exclusively affect the emulator's settings. I also added to `.gitignore` in my last commit.

Reply:
We should also delete it and push the deleted file so it isn't tracked at all

Reply:
Done!

Reply:
Done!
I excluded this file in my last commit.